convert-to-old-roman.js

```diff
@@ -0,0 +1,36 @@
+var pry = require('pryjs');
+
+function convertToOldRoman(number) {
+  numberM = Math.floor(number / 1000);
```

Comment:
Remember when declaring variables to use `var` or `let`/`const` to avoid creating unwanted globals